gradle/dependencies.gradle

```diff
@@ -166,6 +167,8 @@ ext {
 
       svgParser                 : "com.caverock:androidsvg-aar:${version.svgParser}",
 
+      viewBinding               : "com.android.databinding:viewbinding:${version.viewBinding}",
```

Comment:
I see the following warnings with this change:
```
WARNING:Your project has set `android.useAndroidX=true`, but configuration `releaseRuntimeClasspath` still contains legacy support libraries, which may cause runtime issues.
This behavior will not be allowed in Android Gradle plugin 8.0.
Please use only AndroidX dependencies or set `android.enableJetifier=true` in the `gradle.properties` file to migrate your project to AndroidX (see https://developer.android.com/jetpack/androidx/migrate for more info).
The following legacy support libraries are detected:
releaseRuntimeClasspath -> project :libnavui-base -> com.android.databinding:viewbinding:7.2.1
releaseRuntimeClasspath -> project :libnavui-base -> com.android.databinding:viewbinding:7.2.1 -> com.android.support:support-annotations:26.1.0
```

You should have added `androidx.databinding:viewbinding` instead.

Reply:
Fixed in #5914.